docs/BEMAN_STANDARD.md

```diff
@@ -98,7 +98,7 @@ Examples: A `beman.smart_pointer` library's repository should be named `smart_po
 Bad examples: `smartpointer` or `optional26`.
 
 **[REPOSITORY.CODEOWNERS]** REQUIREMENT: There must be a `.github/CODEOWNERS` file
-with a relevant set of codeowners.
+with a relevant set of codeowners. [@bemanproject/beman-codeowner](https://github.com/orgs/bemanproject/teams/beman-codeowners/repositories) must be the first present entry, followed by local codeowners. Please check [exemplar: CODEOWNERS](https://github.com/bemanproject/exemplar/blob/main/.github/CODEOWNERS).
```

Comment:
If we stick with `default-codeowners` as the name for default reviewers.

I suggest leaving a comment here explaining this team is included to speed up review, as it is my personal opinion that the motivation behind this rule might be unclear.

Reply:
I added that comment.